src/main/scala/se/nimsa/dcm4che/streams/DicomParsing.scala

```diff
@@ -20,12 +20,31 @@ import akka.util.ByteString
 import org.dcm4che3.data.{ElementDictionary, VR}
 import org.dcm4che3.io.DicomStreamException
 
+
 /**
   * Helper methods for parsing binary DICOM data.
   */
 trait DicomParsing {
+  val TRANSFER_SYNTAX_IMPLICIT_VR_LITTLE_ENDIAN = "1.2.840.10008.1.2"
```

Comment:
All DICOM UIDs are defined in `org.dcm4che3.data.UID`. I would prefer using those.